src/org/jgroups/protocols/TP.java

```diff
@@ -207,6 +207,9 @@ public String getBundlerClass() {
     public long             getTimeServiceInterval() {return time_service_interval;}
     public <T extends TP> T setTimeServiceInterval(long t) {this.time_service_interval=t; return (T)this;}
 
+    public boolean          useVThreads()          {return use_vthreads;}
+    public <T extends TP> T useVThreads(boolean b) {use_vthreads=b; return (T)this;}
```

Comment:
Also, "VThreads"? That's not a thing to use in public API.

Reply:
It was `use_virtual_threads` in <=5.3 [but was renamed to vthreads in 5.4](https://issues.redhat.com/browse/JGRP-2842?focusedId=25806039&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-25806039)?

Reply:
Updated according to comment.